inkycal/modules/inkycal_agenda.py

```diff
@@ -202,11 +202,15 @@ def generate_image(self):
                     if not parser.all_day(_):
                         write(im_black, (x_time, line_pos[cursor][1]),
                               (time_width, line_height), time,
-                              font=self.font, alignment='left')
+                              font=self.font, alignment='right')
+                    if parser.all_day(_):
```

Comment:
An else statement would be better suited here instead of re-calling the function, causing a little more overhead